code/game/atoms.dm

```diff
@@ -586,6 +587,9 @@ its easier to just keep the beam vertical.
 /atom/proc/singularity_pull()
 	return
 
+/atom/proc/can_emag()
+	return TRUE
```

Comment:
Please confirm you've tested what happens when an area, turf, object without an explicit emag_act, or mob is emagged.

Reply:
nothing happens aside from the attack being totally silent